applications/GeoMechanicsApplication/custom_conditions/U_Pw_face_load_condition.cpp

```diff
@@ -15,6 +15,7 @@
 
 // Application includes
 #include "custom_conditions/U_Pw_face_load_condition.hpp"
+#include "custom_utilities/element_utilities.hpp"
```

Comment:
The functionality used from this is shared between elements and conditions. Should it then still be called element_utilities.

Reply:
Good point. I had the same question when I was making the changes. I would suggest to discuss this in more detail with @rfaasse. Whatever the outcome is, I would propose to delegate that to a separate PR, since it is beyond the scope of this PR.